src/main/webapp/help-initScript.html

```diff
@@ -1,23 +1,33 @@
 <div>
-<b>At a minimum, the init script needs to install a Java runtime</b>.</br>
-
-Custom prepared images are recommended if the initialization script is taking more than 20 minutes to execute.<br></br>
-
-Below are examples of initialization scripts: </br></br>
-1)	<b>Ubuntu</b></br>
-		&nbsp;&nbsp;&nbsp;&nbsp;# Install Java</br>
-		&nbsp;&nbsp;&nbsp;&nbsp;sudo apt-get -y update</br>
-		&nbsp;&nbsp;&nbsp;&nbsp;sudo apt-get install -y openjdk-7-jdk</br>
-		&nbsp;&nbsp;&nbsp;&nbsp;sudo apt-get -y update --fix-missing</br>
-		&nbsp;&nbsp;&nbsp;&nbsp;sudo apt-get install -y openjdk-7-jdk</br></br>
-2)	<b>Windows w/JNLP</b></br>
-    For Windows agents with JNLP launch, this script is a powershell script.</br>
-    You can use this <a target="_blank" href="https://raw.githubusercontent.com/Azure/azure-devops-utils/master/powershell/Jenkins-Windows-Init-Script-no-secrets.ps1">sample</a><br />
-    Automatically passed to this script is:<br/>
-        First argument - Jenkins server URL<br/>
-        Second argument - VMName<br/>
-        Third argument - JNLP secret, required if the server has security enabled.<br/>
-    You need to install Java, download the slave jar file from: '[server url]jnlpJars/slave.jar'.<br/>
-    The server url should already have a trailing slash.  Then execute the following to connect:<br/>
-    java.exe -jar [slave jar location] [-secret [client secret if required]] [server url]computer/[vm name]/slave-agent.jnlp<br/>
+  <p><b>At a minimum, the init script needs to install a Java runtime.</b></p>
+  <p>Custom prepared images are recommended if the initialization script is taking more than 20 minutes to execute.</p>
+  <p>
+    Below are examples of initialization scripts:
+    <ol>
+      <li>
+        <b>Ubuntu</b>
+        <pre><code># Install Java
+sudo apt-get -y update
+sudo apt-get install -y openjdk-7-jdk
+sudo apt-get -y update --fix-missing
+sudo apt-get install -y openjdk-7-jdk
```

Comment:
I just tried seems the last two lines are not necessary. @clguimanMSFT is there special reason to run update with --fix-missing and install java again?